File: core-java-practice/gcr-codebase/VolumeOfCylinder.java

import java.util.Scanner;

public class VolumeOfCylinder{
      public static void main(String[] args) {
        // read input
        Scanner sc=new Scanner(System.in);

        System.out.print("Enter the radius of the cylinder: ");
        int radius=sc.nextInt();  //take input

        System.out.print("Enter the height of the cylinder: ");
        int height=sc.nextInt();  //take input 

        //formula to calculate the volume of cylinder
        double volume=Math.PI * Math.pow(radius,2) * height;

        //printing the volume 
        System.out.println("The volume of the cylinder is :" + volume);

      }
}

File: core-csharp-practice/gcr-codebase/programming-elements/VolumeOfCylinder.cs
using System;

class VolumeOfCylinder{
    public static void Main(string[] args){

        Console.Write("Enter the radius of the cylinder: ");
        int radius=int.Parse(Console.ReadLine()); //taking radius as an input

        Console.Write("Enter the height of the cylinder: ");
        int  height=int.Parse(Console.ReadLine()); //taking height as an input

        //volume calculation
        double volume=Math.PI * Math.Pow(radius,2) * height;

        Console.Write("The volume of the cylinder is: " + volume);
    }
}